slsim/lens.py

```diff
@@ -502,6 +531,23 @@ def point_source_magnitude_molet(self, band, time, **kwargs_molet):
 
         # TODO: in what format should be the 2d source profile be stored (as it is time- and wavelength dependent)
         # TODO: do we create full light curves (and save it in cache) or call it each time
+
+        magnitude_without_microlensing = self._point_source_magnitude(
```

Comment:
can you make a function that checks whether the interpolation exists and if so, calls the interpolation function to get the correct format back?
And then also define what you need from the molet return to make it an interpolation?
Thank you!